.github/workflows/execution-environments.yml

```diff
@@ -0,0 +1,45 @@
+name: execution environment images
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  # Run once per week (Monday at 08:00 UTC)
+  schedule:
+    - cron: '0 8 * * 1'
+
+jobs:
+  build:
+    name: 'Build EE image: ${{ matrix.name }}'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - name: 2.12-with-ansible5
```

Comment:
Out of curiosity: wouldn't it make more sense to simply call it `ansible5`? Since we see ansible-core 2.12 as a part of Ansible 5, it is clear from 'Ansible 5' that it contains ansible-core 2.12.